docs/howto/gathering_info/mission_impact.md

```diff
@@ -0,0 +1,16 @@
+# Gathering Information About Mission Impact
+
+```python exec="true" idprefix=""
+from ssvc.decision_points.ssvc.mission_impact import LATEST
+from ssvc.doc_helpers import example_block
+
+print(example_block(LATEST))
+```
+
+The factors that influence the mission impact level are diverse.
+The material here does not exhaustively discuss how a stakeholder should answer a question; that is a topic for future work.
```

Comment:
`The material here...future work.` made sense in the paper, it's out of place here. I think you can just delete the line.